Integrations/FeedOffice365/FeedOffice365.yml

```diff
@@ -3,6 +3,11 @@ commonfields:
   id: FeedOffice365
```

Comment:
```suggestion
  id: Office365 Feed
```

Reply:
Need to update wherever you reference the integration ID as well such as in the `conf.json` or whatnot.